include/tacho_motor_class.h

```diff
@@ -194,9 +194,4 @@ extern void unregister_tacho_motor(struct tacho_motor_device *);
 
 extern struct class tacho_motor_class;
 
-inline struct tacho_motor_device *to_tacho_motor(struct device *dev)
```

Comment:
OK, well this works so lets leave it. I should have checked for other `static inline` usage, but it never feels right to declare (not define) in a `.h` file...